hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -252,7 +252,7 @@ func (r *HostedClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	err := r.Get(ctx, req.NamespacedName, hcluster)
 	if err != nil {
 		if apierrors.IsNotFound(err) {
-			log.Error(err, "hostedcluster not found, aborting reconcile", "name", req.NamespacedName)
+			log.Info("hostedcluster not found, aborting reconcile", "name", req.NamespacedName)
```

Comment:
Now I wonder why even log this at all?

Reply:
If it's totally normal and non actionable it's just noise

Reply:
It can show issues, for example the lack of cleanup of a clusterrolebinding we had, which triggered reconciliations of clusters that were long gone

Reply:
If that's a generalizable regression we want to guard against I wonder if there's a missing metric... does it seem true that in the normal case we'd only ever really expect very few (1 or 2) reconcile attempts of a deleted cluster? If so that could be tracked as a metric and a budget placed around it? etc.